CONTRIBUTING.md

```diff
@@ -2,10 +2,41 @@
 
 Thanks for contributing to the VIP Javascript library. There are some guidelines to ensure we have consistency across the CLI and web interfaces.
 
-## Adding commands
+## Coding Standards
+
+TODO
+
+## Developing
+
+### args package
+
+TODO
+
+### Adding commands
 
 * New command names should use the singular case (e.g. site vs sites)
 
-## Adding libraries
+### Adding libraries
 
 New libraries should generally support both CLI and web contexts, though some cases that won't make sense (e.g. formatting for CLI output). Ensuring the libraries are useful everywhere will allow us to offer consistent experiences regardless of the interface.
+
+## Releasing
+
+Note: We use `publish-release` for some pre-publish confidence checks to avoid common mistakes.
+
+### Major / Minor Versions
+
+1. Set the version: `npm version minor` or `npm version major`
```

Comment:
You most likely usually want `npm version patch` or `npm version minor`. I suspect it will be a while before we do a 2.0.0.